Project.toml

```diff
@@ -0,0 +1,25 @@
+name = "ECOS"
+uuid = "e2685f51-7e38-5353-a97d-a921fd2c8199"
+repo = "https://github.com/JuliaOpt/ECOS.jl.git"
+version = "0.9.4"
```

Comment:
We ought to be bumping the minor version number when we drop support for julia releases.